### PR TITLE
Feature/final touch

### PR DESCRIPTION
**Pull request by Zafron047**
Dear Team,

I hope this message finds you well. I am pleased to present the latest updates for the "feature/final-touch" branch. These changes bring a more professional and polished appearance to the project. Here's a summary of the enhancements:

1. **Button Hover Effect:** Improved the visual appeal of buttons by introducing a glossy hover effect. The buttons now gracefully transition to a slightly lighter shade of green with a subtle box-shadow, creating a more engaging user experience.

2. **Updated Styling:** Made consistent changes to border colors across various CSS files (Profile.css, nav.css, RocketList.css, Mission.css) to ensure a cohesive design throughout the application.

3. **Readme Refinement:** Revamped the Readme file to provide comprehensive project information. Added a well-structured Table of Contents for easy navigation and highlighted essential sections, such as "About the Project," "Tech Stack," "Live Demo," "Getting Started," "Contributing," and more.

4. **Live Demo Link:** Incorporated a live demo link, allowing stakeholders to access and experience the project effortlessly.

I kindly request the team's esteemed review and consideration for merging the "feature/final-touch" branch into the `Dev` repository. These updates significantly enhance the project's professionalism and user experience.

Thank you for your time and collaboration in making this project exceptional!

Warm regards,
@Zafron047

### CODE REVIEW

File: src/components/styles/Profile.css [hunk 1]
```diff
@@ -1,14 +1,38 @@
-.profile {
+.profile-container {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 8rem;
+}
```
Comment (zuhebahmed88091) [inline src/components/styles/Profile.css:5]: Kindly fix the design:
->The profile-container height should be 100%.
Please check the image below->

ss1

File: src/components/styles/Nav.css
Comment (zuhebahmed88091) [inline src/components/styles/Nav.css:1]: Kindly check the image. And fix the Space Travelers Hub header. It's breaking. 

ss2

**Review by zuhebahmed88091 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Zafron047 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: The structure of the files  follow the `ducks pattern`
:heavy_check_mark: Use of correct Gitflow
:heavy_check_mark: Well documented readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/Profile.css (hunk 1 above)
Comment (Zafron047) [reply src/components/styles/Profile.css]: Resolved with min-height: 100vh.

File: src/components/styles/Nav.css
Comment (Zafron047) [reply src/components/styles/Nav.css:1]: Issue resolved by making the font sizes responsive.

**Comment by Zafron047** [comment]
@zuhebahmed88091 I have made the necessary changes. Please kindly check the changes. 😸

**Review by zuhebahmed88091 — APPROVED** [review]
Hi @Zafron047 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Commit messages meaningful ✔️
- Clean code ✔️
- All functionalities in place ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zafron047** [comment]
@zuhebahmed88091 Thanks for the review and your valuable suggestions.

**Pull request merged**